docs/e2e_flows.md

```diff
@@ -0,0 +1,22 @@
+# E2E Flows
+
+## Data sent from application to CKAN through Data API 
+In order to send data from your application to CKAN through the Data API you need to follow the next steps:
+
+### Pre-requisities:
+- Ask for a service to the platform administrator.
+- Ask for an admin user for your service to the platform administrator.
+- Ask for a CKAN user for your service to the platform administrator.
+- Ask for an organization in CKAN to the platform administrator. The organization name should be the name of your service.
+- Ask for a dataset in CKAN to the platform administrator. The name of the dataset should be "yourservice_yoursubservice". If you don't have a subservice the name of the dataset should be the name of the service.
+- Ask for a resource in CKAN to each one of your entities to the platform administrator. The name of your resources should be "entityname_entitytype". 
+You can also use a single resource for a group of entities using *name mappings* (see XXX)
+- Ask for a datastore for each resource in CKAN to the platform administrator. The columns of the datastore should be the name of your entity attributes. The type of the columns should be *json*. There should be also the following mandatory columns:_id, recvTime, fiwareServiePath, entityType, entityId and TimeInstant. Also it is needed to have a column of metadatas for each attribute named "entityattribute_md"
+- Create a subservice (not mandatory)
```

Comment:
Similarly, you could have a "Create" bullet at first level, then the different elements to create as sub-items.